microsoft-edge/devtools-guide-chromium/workspaces/index.md

```diff
@@ -176,69 +198,68 @@ The DOM tree is not the HTML source tagging.
-## Step 4: Save a JavaScript change to disk
+#### Save a JavaScript change to disk
 
 The main place to use the code editor of DevTools is the **Sources** tool.  But sometimes you need to access other tools, such as the **Elements** tool or the **Console** panel, while editing files.  The **Quick source** tool gives you just the editor from the **Sources** tool, while any tool is open.
 
 To open the DevTools code editor alongside other tools:
 
-1. Navigate to the **Elements** tool.
+1. Select the **Elements** tool.
 
-1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS).  The **Command Menu** opens.
+1. Press `Ctrl`+`Shift`+`P` (Windows, Linux) or `Command`+`Shift`+`P` (macOS) to open the **Command Menu**.
 
-1. Type `quick`, and then select **Show Quick source**.  At the bottom of the DevTools window, the **Quick source** tool appears, displaying the contents of `index.html`, which is the last file you edited in the **Sources** tool.
+1. Type `quick` at the **Run** prompt, and then select **Show Quick source**.  At the bottom of the DevTools window, the **Quick source** tool appears, displaying the contents of `index.html`, which is the last file you edited in the **Sources** tool.
 
    ![Open the 'Quick source' tool by using the Command Menu.](../media/workspaces-workspaces-demo-search-show-quick-source.msft.png)
 
-1. Press `Ctrl`+`P` (Windows, Linux) or `Command`+`P` (macOS) to open the **Open File** dialog, as shown below.
+1. Press `Ctrl`+`P` (Windows, Linux) or `Command`+`P` (macOS) to open the **Open File** dialog, shown in the next screenshot.
 
-1. Type `script`, then select **app/script.js**.
+1. Type `script` at the **Open** prompt, then select **app/script.js**.
 
    ![Open script.js using the Open File dialog.](../media/workspaces-workspaces-demo-search-script.msft.png)
 
    > [!NOTE]
    > The `Save Changes To Disk With Workspaces` link in the demo is styled regularly.
 
-1. Add the following code to the bottom of **script.js** using the **Quick source** tool.
+1. Use the **Quick source** tool to add the following code to the bottom of **script.js**.
 
     ```javascript
-    console.log('greetings from script.js');
     document.querySelector('a').style = 'font-style:italic';
     ```
 
 1. Press `Ctrl`+`S` (Windows, Linux) or `Command`+`S` (macOS) to save the change.
 
 1. Refresh the page.  The link on the page is now italicized.
 
-![The link on the page is now italicized.](../media/workspaces-workspaces-demo-elements-styles-quick-source-script.msft.png)
+   ![The link on the page is now italicized.](../media/workspaces-workspaces-demo-elements-styles-quick-source-script.msft.png)
 
 
 <!-- ====================================================================== -->
 ## Next steps
 
-Use what you have learned in this tutorial to set up Workspaces in your own project.
+Use what you’ve  learned in this tutorial to set up and use DevTools workspaces in your own project.
```

Comment:
```suggestion
Use what you've  learned in this tutorial to set up and use DevTools workspaces in your own project.
```

Reply:
https://learn.microsoft.com/en-us/contribute/docs-authoring/smart-quote-replacement
One technique is, if you have tools set up some way:
Select all, in .md in VS Code.
Cut.
Paste.
Save.
Check diff in GitHub Desktop to see where quotes were straightened.

Reply:
It's a matter of checking my copy and paste from preliminary work in my word doc, I'll be more rigorous going forward.